Source/NSTextView.m

```diff
@@ -4982,6 +4982,40 @@ the range of characters (if any) to be replaced by the new data.  */
 	  RELEASE(attachment);
 	  return YES;
 	}
+      if ([type isEqualToString: NSFilenamesPboardType])
+	{
+          NSArray *list = [pboard propertyListForType: NSFilenamesPboardType];
+          NSMutableAttributedString *as = [[NSMutableAttributedString alloc] init]; 
+
+	  for (NSString *filename in list)
+	   {
+	      NSFileWrapper *fw = [[NSFileWrapper alloc] initWithPath:filename];
+	      if (fw) 
+	        {
+	          NSTextAttachment *attachment = [[NSTextAttachment alloc] 
+					         initWithFileWrapper: fw];
+	          NSAttributedString *asat =
+	            [NSAttributedString attributedStringWithAttachment: attachment];
+
+	          RELEASE(fw);
+	          RELEASE(attachment);
+
+	          [as appendAttributedString:asat];
```

Comment:
Here a space after the colon is missing.

Reply:
done